tests/conftest.py

```diff
@@ -81,10 +81,14 @@ def catch_stdout():
 
 @pytest.fixture
 def patch_logger(silent=True):
-    new_logger = Logger(silent)
+    class PatchedLogger(Logger):
+        def info(self, msg, *args, prefix=None, extra=None, exc_info=None):
+            super(PatchedLogger, self).info(msg, *args)
 
     from connectors.logger import logger
 
+    new_logger = PatchedLogger(silent)
+
```

Comment:
It goes deeper. We have a couple of tests that verify some logs entries and for that we need to have patched logger.  My guess that someone patched only `info`  and redirect all the call to `info` to avoid patching all the logger methods. 

I personally don't like this approach and I didn't want to spend time refactoring this.